pkg/resource/generator.go

```diff
@@ -284,6 +286,41 @@ func (g *Generator) Remove(cr *imageregistryv1.Config, modified *bool) error {
 		glog.Infof("object %s deleted", Name(gen))
 		*modified = true
 	}
+	rclient, err := regopset.NewForConfig(g.kubeconfig)
+	if err != nil {
+		return err
+	}
+	driver, err := storage.NewDriver(cr.ObjectMeta.Name, cr.ObjectMeta.Namespace, &cr.Status.Storage)
+	if err != nil {
+		return err
+	}
+
+	o := cr
```

Comment:
not seeing the point of this either... looks like you can just keep using the cr variable

Reply:
ok, nm, i do see why you need the copy.  still don't think you need the nil assignment below though.

Reply:
updated